lib/Pod/Perldoc.pm

```diff
@@ -594,11 +598,10 @@ sub init_formatter_class_list {
   # Only those will work propertly with ToMan
   # The rest is either ToTerm or ToMan again
   if ( my $roffer = $self->{'executables'}{'nroffer'} ) {
-    my $minimum_groff_version = '1.20.1';
-    my $version_string        = `$roffer -v`;
+    my $version_string = `$roffer -v`;
     my( $version ) = $version_string =~ /\(?groff\)? version (\d+\.\d+(?:\.\d+)?)/;
 
-    $version ge $minimum_groff_version
+    $version ge MIN_GROFF_VERSION()
```

Comment:
Should this be a string comparison since they are using semantic versioning (or any versioning)?

Reply:
I'll rewrite this to do a proper semver check.

Reply:
Pushed a commit that adds a proper semver check for ngroff with tests.